enhancements/monitoring/scrape-profiles.md

```diff
@@ -0,0 +1,407 @@
+function properly: alerting rules, console dashboards, resource metrics API,
+horizontal/vertical pod autoscaling and Red Hat Telemetry. Users also are not
+allowed to tune the interval at which Prometheus scrapes targets as this again
+can have unforeseen results that can hinder the platform: a low scrape interval
+value may overwhelm the platform Prometheus instance while a high interval value
+may render some of the default alerts ineffective.
+
+The goal of this proposal is to allow users to pick their desired level of
+scraping while limiting the impact this might have on the platform, via
+resources under the control of the cluster-monitoring-operator and other
+platform operators.
+
+Furthermore to assess the viability of the scrape profile feature the monitoring
+team performed a detailed analysis of its impact in an OpenShift cluster. The
+analysis consisted in a test where an OpenShift cluster would run three replicas
+of the OpenShift Prometheus instance but each replica would be configured to a
+different scrape profile (`full`, `minimal`). Then we would trigger a
+workload using kube-burner and at the end of 2 hours, we evaluated the results.
+We concluded that in terms of resource usage, given the results obtained we can
+confidently state that the feature is quite valuable given the reduction of CPU
+by ~21% and memory by ~33% when comparing the `minimal` to the `full` profile.
+More detail can be consulted in this
+[document](https://docs.google.com/document/d/1MA-HTJQ_X7y_bwpJS2IPbGmC4qDyIMp25jisr34X2F4/edit?usp=sharing)
+
+Moreover, through Telemetry, we collect for each cluster the top 3 Prometheus
+jobs that generate the biggest amount of samples. With this data, we know that
+for OpenShift 4.11 the 5 components most often reported as the biggest producers
+are: the Kubernetes API servers, the Kubernetes schedulers, kube-state-metrics,
+kubelet and the network daemon.
+
+
+### User Stories
+
+- As an OpenShift cluster administrator, I want to lower the amount of resources
+  consumed by Prometheus in a supported way, so I can choose between different
+  scrape profiles, e.g `full` or `minimal`.
+- As an OpenShift developer, I want a supported way to collect a subset of the
+  metrics exported by my operator and operands.
+
+### Goals
+
+- Give users a supported and documented method to pick the amount of metrics
+  being collected by the platform, in a way that allows
+  cluster-monitoring-operator and other operators to provide their own
+  defaulting.
+- Other components of the cluster (console, HPA, alerts) should not be
+  negatively affected by this change.
+- Openshift developers may optionally adhere to this change, developers that do
+  not adhere should not be impacted.
+
+### Non-Goals
+
+- Dynamically adjusting metrics scraped at runtime based on heuristics.
+- Allowing users to explicitly set a list or a regex of metrics that they would
+want to be included in a given profile.
```

Comment:
IIUC the second case [here](https://issues.redhat.com/browse/MON-2694?focusedId=21834890&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-21834890) suggests otherwise?

Reply:
Here with `user` I mean OpenShift end users while in the link you provided I mean OpenShift developer